homeassistant/components/homekit/accessories.py

```diff
@@ -0,0 +1,55 @@
+"""Extend the basic Accessory and Bridge functions."""
+from pyhap.accessory import Accessory, Bridge, Category
+
+from .const import (
+    SERVICES_ACCESSORY_INFO, MANUFACTURER,
+    CHAR_MODEL, CHAR_MANUFACTURER, CHAR_SERIAL_NUMBER)
+
+
+class HomeAccessory(Accessory):
+    """Class to extend the Accessory class."""
+
+    ALL_CATEGORIES = Category
+
+    def __init__(self, display_name):
+        """Initialize a Accessory object."""
+        super().__init__(display_name)
+
+    def set_category(self, category):
+        """Set the category of the accessory."""
+        self.category = category
+
+    def set_services(self, services):
+        """Define the services to be available for the accessory."""
+        from pyhap.loader import get_serv_loader
+        self.add_service(get_serv_loader().get(services))
```

Comment:
loader gets a single Service, maybe remove plural?

Reply:
The idea was to handle all service loading from this method. Maybe later extend the functionality to support lists as input, hence `services`. For know `service` should be enough.